modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/configurators/TargetPlatformRootProjectConfigurator.java

```diff
@@ -53,14 +56,48 @@ public void apply(final Project project) {
 			return;
 		}
 
+		Matcher matcher = _externalVersionPattern.matcher(
+			targetPlatformVersion);
+
+		final String repositoryVersion;
+
+		if (matcher.matches()) {
+			StringBuilder sb = new StringBuilder();
+
+			sb.append(matcher.group(1));
+			sb.append('.');
+			sb.append(matcher.group(2));
+			sb.append('.');
+
+			String label = matcher.group(3);
+
+			try {
+				int labelNumber = Integer.parseInt(matcher.group(4));
+
+				if (label.startsWith("GA")) {
+					sb.append(labelNumber - 1);
+				}
+				else if (label.startsWith("sp")) {
+					sb.append(labelNumber);
+				}
+			}
+			catch (NumberFormatException nfe) {
+			}
```

Comment:
I know SF complained about this, shouldn't we rethrow it, maybe with `throw new GradleException(nfe)`?

Reply:
Ok will do